core/services/relay/evm/mercury/transmitter.go

```diff
@@ -54,17 +49,23 @@ func getPayloadTypes() abi.Arguments {
 	})
 }
 
-func NewTransmitter(lggr logger.Logger, httpClient HTTPClient, fromAccount common.Address, reportURL, username, password string) *MercuryTransmitter {
-	return &MercuryTransmitter{lggr.Named("Mercury"), httpClient, fromAccount, reportURL, username, password}
+func NewTransmitter(lggr logger.Logger, rpcClient wsrpc.Client, fromAccount common.Address, reportURL, username, password string) *mercuryTransmitter {
+	return &mercuryTransmitter{lggr.Named("Mercury"), rpcClient, fromAccount, reportURL, username, password}
+}
+
+func (mt *mercuryTransmitter) Start(ctx context.Context) error {
+	return mt.rpcClient.Start(ctx)
 }
 
-type MercuryReport struct {
-	Payload     hexutil.Bytes
-	FromAccount common.Address
+func (mt *mercuryTransmitter) Close() error {
+	return mt.rpcClient.Close()
 }
 
+func (mt *mercuryTransmitter) Healthy() error { return nil } //TODO
+func (mt *mercuryTransmitter) Ready() error   { return nil } //TODO
```

Comment:
Will these be included in this PR? Maybe a `utils.StartStopOnce` would be worth it.

Reply:
Oh wait does the `wsrpc.Client` already have one? Embedding the `*wsrpc.Client` field in the struct to inherit all four methods would be simpler in that case.

Reply:
Opted to not embed just to avoid confusion about the `Transmit` function.